docs/docs-content/clusters/edge/local-ui/host-management/custom-link.md

```diff
@@ -64,31 +64,37 @@ or open a new tab in your browser depending on how to configure the links.
    git checkout v4.4.0
    ```
 
-4. Create a directory called **ui** at the path `CanvOS/ui`.
+4. Create a directory called **local-ui** at the path **CanvOS/local-ui**. Then under the **local-ui** directory, create
+   a directory named **ui**.
 
-5. In the directory, create a file named **customizations.json**. This is the same file where you customize the theme of
-   Local UI. If the file is already present, there is no need to create a new one. For more information about
-   customizing the theme of Local UI, refer to [Customize Local UI Theme](./theming.md).
+5. In the **CanvOS/local-ui/ui** directory, create an file named **customizations.json**. This is the same file where
+   you customize the theme of Local UI. If the file is already present, there is no need to create a new one. For more
+   information about customizing the theme of Local UI, refer to [Customize Local UI Theme](./theming.md).
 
-6. If you want Local UI to host static web pages that you want your users to be able to access from the sidebar, put the
-   HTML files in the **ui** folder. The HTML file may link to JavaScript and CSS files. Ensure that your HTML file can
-   reference its assets. We suggest that you put all assets for specific web pages in one folder. The hosted static
-   sites are exposed through HTTPS with a self-signed certificate.
+6. If you want Local UI to host static web pages that you want your users to be able to access from the sidebar, create
+   another folder for your website under the **local-ui** folder and put all assets for your web pages in that folder.
 
-   For example, you can put the following content inside the **ui** folder.
+   Choose a name that describes the web pages. For example, if your website has content related to an application, you
+   can name the folder **app** and create the folder at **local-ui/app**.
+
+   The HTML files may link to JavaScript and CSS files. Ensure that your HTML file can reference its assets. The hosted
+   static sites are exposed through HTTPS with a self-signed certificate.
+
+   For example, you can put the following content inside the **local-ui** folder.
 
    ```text
-   |-ui
-     |--customizations.json
+   |-local-ui
+     |--ui
+         |--customizations.json
      |--app
           |--index.html
           |--index.js
           |--index.css
    ```
 
-   When using an URL to locate the local webpage, you can treat the **ui** folder as the root folder and you must use
-   absolute paths to locate your assets. For example, you can locate the HTML file in the example with
-   `/app/index.html`.
+   When using an URL to locate the local webpage, you can treat the **local-ui** folder as the root folder and you must
+   use absolute paths to locate your assets. For example, from **customizations.json**, you can locate the HTML file in
+   the example with `/app/index.html`.
```

Comment:
from customizations.json this should be ../app/index.html ?

Reply:
So I know that you tested that relative links work, but when I used relative links last time, @codrin-iftimie specifically told me that i had to use absolute links. So I used absolute links here. 

@codrin-iftimie Can you confirm that either absolute or relative link will work? Can you tell us why you didn't want me using relative links before?

Reply:
Since "app" was under "ui" - ui/app/index.html in 4.4, should this be highlighted as a breaking change for existing 4.4 customers using custom-links ?
Someway to indicate that the custom apps are now expected to be at the same level as the "ui" folder instead of being a subfolder, under ui/, when the 4.4 env is upgraded to 4.4a.

Reply:
Yes, this is marked as a breaking change. We document it in the release notes PR [https://github.com/spectrocloud/librarium/pull/3320](Release Notes )

Reply:
If you see the left side(old) structure, customizations.json and app are in the same folder, so /app/index.html would work. Now app has moved one level up, so we need ../app/index.html as the relative path from customization.json.

Reply:
With absolute paths though, isn't **local-ui** the root folder, and therefore it'd be **/app/index.html** if it's absolute path??

Reply:
I am going to change this to relative path for now with your suggestion. I do want to hear from Codrin on why he wanted me to use absolute path before merging the PR. I'll leave him a message and merge this when I hear back.

Reply:
There are two types of references the customer would use:
1. the link of the application which is something provided in `customizations.json` under  `"link"` attribute for each entry. This needs to be absolute as Lenny mentioned. It can't be relative because the user would access this from the menu from different URLs (eg: on cluster details page `<IP>/cluster/details` we'd append the relative URL which would resolve in a 404 `<IP>/cluster/details/app/index.html`)
2. the applications will have resources that they will reference inside the `index.html` file. This is up to the developer that does this. The good practice is to use absolute URLs paths as well, especially if it's a SPA (single page application)